### ISSUE TITLE
Not wild about the menu structure

### ISSUE THREAD

**Issue by geoidesic** [dreamonkey/quasar#125]
**Description**
_Clearly and concisely explain the problem.
You can mention unclear or missing information, typos, grammatical mistakes, etc.
Any contributions and suggestions are welcome!_

I was missing my usual vertical left hand side menu for components etc. Had to guess that I must click on "Why Quasar?" in the horizontal menu to find it. Not ideal. Personally I preferred the old style with the menu always available on the left hand side. Why do you think it's better to have two menus?

**Website Section URL**
_[Website live preview](https://quasar.beget.it), paste the page URL or section name (eg. Docs, if it happens on all docs page) below._

Home page

**Comment by IlCallo** [comment]
Sorry, could you expand on where you find trouble to reach components?
You wrote "homepage" but it seems you're talking about the "landing" page

There is now again a separation between the 2 pages, like there were before the migration to Qv2
From the landing page, you can reach the actual docs (with the open drawer by default) using pretty much all links in the header

**Comment by geoidesic** [comment]
I'm not sure of the difference between home page and landing page. I simply followed the link in the Discord announcement... wherever that led is the page I meant.

My eye was not drawn to the links in the header. They were drawn to the links below that, probably because they are wider (i.e. as a whole take up more space) and right next to the outlined version selector which is actually what drew my eye first. I only noticed the links at the top after browsing for a bit.

So I naturally clicked the first link in the menu I was looking at, which was `Why Quasar?`, which then provided the left hand menu of components etc. So I deduced from this experience that the place in which the documentation was stored was behind the `Why Quasar?` link, which seemed odd. 

However, now that I'm aware of the "Docs" link it makes more sense. It was a confusing experience for me though.
